bin/client.py

```diff
@@ -278,25 +278,42 @@ def main():
     install_exc_handler(default_handler)
     ver = 'APEL client %s.%s.%s' % __version__
     opt_parser = OptionParser(version=ver, description=__doc__)
+    default_conf_location = '/etc/apel/client.cfg'
```

Comment:
As these are constants, they should be in ALL_CAPS.
https://peps.python.org/pep-0008/#constants

Reply:
done